terraform/environment/regions.tf

```diff
@@ -12,6 +12,11 @@ module "allow_list" {
   source = "git@github.com:ministryofjustice/opg-terraform-aws-moj-ip-allow-list.git?ref=v2.3.0"
 }
 
+data "aws_opensearchserverless_collection" "lpas_collection" {
```

Comment:
we shouldn't use the data source here because it will look for it at the refresh stage before it's created and will fail. instead use the attribute of the resource `aws_opensearchserverless_collection.lpas_collection.collection_endpoint`